bugbug/models/duplicate.py

```diff
@@ -32,7 +28,10 @@ def fit(self, X, y):
 
 
 class DuplicateModel(BugCoupleModel):
-    def __init__(self, lemmatization=False):
+    def __init__(self, training_size, lemmatization=False):
+        self.NUM_DUPLICATES = training_size // 2
+        self.NUM_NONDUPS_NONDUPS = self.NUM_DUP_NONDUPS = training_size // 4
```

Comment:
In Python, the convention is to use uppercase for constants. Since these are not constants anymore, make them lower case.